cime_config/config_pes.xml

```diff
@@ -112,1282 +112,277 @@
+	        <ntasks_cpl>-4</ntasks_cpl>
+	      </ntasks>
+	      <nthrds>
+	        <nthrds_atm>2</nthrds_atm>
+	        <nthrds_lnd>2</nthrds_lnd>
+	        <nthrds_rof>2</nthrds_rof>
+	        <nthrds_ice>2</nthrds_ice>
+	        <nthrds_ocn>2</nthrds_ocn>
+	        <nthrds_glc>2</nthrds_glc>
+	        <nthrds_wav>2</nthrds_wav>
+	        <nthrds_cpl>2</nthrds_cpl>
+	      </nthrds>
+	      <rootpe>
+	        <rootpe_atm>0</rootpe_atm>
+	        <rootpe_lnd>0</rootpe_lnd>
+	        <rootpe_rof>0</rootpe_rof>
+	        <rootpe_ice>0</rootpe_ice>
+	        <rootpe_ocn>0</rootpe_ocn>
+	        <rootpe_glc>0</rootpe_glc>
+	        <rootpe_wav>0</rootpe_wav>
+	        <rootpe_cpl>0</rootpe_cpl>
+	      </rootpe>
       </pes>
     </mach>
   </grid>
   <grid name="a%ne240">
     <mach name="any">
       <pes pesize="any" compset="any">
-	<comment>none</comment>
-	<ntasks>
-	  <ntasks_atm>-32</ntasks_atm>
-	  <ntasks_lnd>-32</ntasks_lnd>
-	  <ntasks_rof>-32</ntasks_rof>
-	  <ntasks_ice>-32</ntasks_ice>
-	  <ntasks_ocn>-32</ntasks_ocn>
-	  <ntasks_glc>-32</ntasks_glc>
-	  <ntasks_wav>-32</ntasks_wav>
-	  <ntasks_cpl>-32</ntasks_cpl>
-	</ntasks>
-	<nthrds>
-	  <nthrds_atm>1</nthrds_atm>
-	  <nthrds_lnd>1</nthrds_lnd>
-	  <nthrds_rof>1</nthrds_rof>
-	  <nthrds_ice>1</nthrds_ice>
-	  <nthrds_ocn>1</nthrds_ocn>
-	  <nthrds_glc>1</nthrds_glc>
-	  <nthrds_wav>1</nthrds_wav>
-	  <nthrds_cpl>1</nthrds_cpl>
-	</nthrds>
-	<rootpe>
-	  <rootpe_atm>0</rootpe_atm>
-	  <rootpe_lnd>0</rootpe_lnd>
-	  <rootpe_rof>0</rootpe_rof>
-	  <rootpe_ice>0</rootpe_ice>
-	  <rootpe_ocn>0</rootpe_ocn>
-	  <rootpe_glc>0</rootpe_glc>
-	  <rootpe_wav>0</rootpe_wav>
-	  <rootpe_cpl>0</rootpe_cpl>
-	</rootpe>
-      </pes>
-    </mach>
-  </grid>
-  <grid name="a%T31">
-    <mach name="hera|sierra" >
-      <pes pesize="any" compset="any">
-	<comment>none</comment>
-	<ntasks>
-	  <ntasks_atm>64</ntasks_atm>
-	  <ntasks_lnd>64</ntasks_lnd>
-	  <ntasks_rof>64</ntasks_rof>
-	  <ntasks_ice>64</ntasks_ice>
-	  <ntasks_ocn>64</ntasks_ocn>
-	  <ntasks_glc>64</ntasks_glc>
-	  <ntasks_wav>64</ntasks_wav>
-	  <ntasks_cpl>64</ntasks_cpl>
-	</ntasks>
-	<nthrds>
-	  <nthrds_atm>1</nthrds_atm>
-	  <nthrds_lnd>1</nthrds_lnd>
-	  <nthrds_rof>1</nthrds_rof>
-	  <nthrds_ice>1</nthrds_ice>
-	  <nthrds_ocn>1</nthrds_ocn>
-	  <nthrds_glc>1</nthrds_glc>
-	  <nthrds_wav>1</nthrds_wav>
-	  <nthrds_cpl>1</nthrds_cpl>
-	</nthrds>
-	<rootpe>
-	  <rootpe_atm>0</rootpe_atm>
-	  <rootpe_lnd>0</rootpe_lnd>
-	  <rootpe_rof>0</rootpe_rof>
-	  <rootpe_ice>0</rootpe_ice>
-	  <rootpe_ocn>0</rootpe_ocn>
-	  <rootpe_glc>0</rootpe_glc>
-	  <rootpe_wav>0</rootpe_wav>
-	  <rootpe_cpl>0</rootpe_cpl>
-	</rootpe>
+	      <comment>none</comment>
+	      <ntasks>
+	        <ntasks_atm>-32</ntasks_atm>
+	        <ntasks_lnd>-32</ntasks_lnd>
+	        <ntasks_rof>-32</ntasks_rof>
+	        <ntasks_ice>-32</ntasks_ice>
+	        <ntasks_ocn>-32</ntasks_ocn>
+	        <ntasks_glc>-32</ntasks_glc>
+	        <ntasks_wav>-32</ntasks_wav>
```

Comment:
Wave can run on a large number of tasks?

Reply:
You will not have a wave component in a cam defined compset. The config_pes in the cam cime_config is just for F and Q compsets - so the wav will always be SWAV.